### PR TITLE
feat: adds a brand new delete button

### PR DESCRIPTION
**Pull request by faouziH21**
this adds a delete button that deletes all drawings.

### CODE REVIEW

File: src/components/ToolMenu/Draw/DeleteAllButton/index.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      icon: <ExclamationCircleOutlined />,
+      onOk(){
+
+        layers[0].getSource()?.clear();
+
+      }
+
+    });
+
+  };
```
Comment (dnlkoch) [inline src/components/ToolMenu/Draw/DeleteAllButton/index.tsx:80]: Very minor, but the extra new lines can be removed.

File: src/components/ToolMenu/Draw/DeleteAllButton/index.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+  if (!layers) {
+    return null;
+  }
+
+  const onClick = () => {
+
+    confirm({
+      title: (t('Draw.deleteAll')),
+      content: (t('Draw.confirmMessage')),
```
Comment (dnlkoch) [inline src/components/ToolMenu/Draw/DeleteAllButton/index.tsx:70]: The locale should be placed inside the corresponding locale object: `Draw` -> `DeleteAllButton`.

File: src/components/ToolMenu/Draw/index.tsx [hunk 3]
```diff
@@ -345,12 +345,25 @@ export const Draw: React.FC<DrawProps> = ({
           </span>
         </DeleteButton>
       ) : <></>}
+      {showDeleteFeatures ? (
+        <DeleteAllButton
+          name = "draw-delte-all"
```
Comment (dnlkoch) [inline src/components/ToolMenu/Draw/index.tsx:350]:
```suggestion
          name="draw-delete-all"
```

File: src/components/ToolMenu/Draw/index.tsx [hunk 4]
```diff
@@ -345,12 +345,25 @@ export const Draw: React.FC<DrawProps> = ({
...
+            className="draw-delete-all"
+          >
+            {t('Draw.deleteAll')}
+          </span>
+        </DeleteAllButton>
+      ):<></>}
 
-      {showStyleFeatures ? (
-        <StylingButton />
-      ): <></>}
```
[the remark this answers is not in the document]
Comment (faouziH21) [reply src/components/ToolMenu/Draw/index.tsx:351]: no should be fixed in: [e37a796](https://github.com/terrestris/shogun-gis-client/pull/1476/commits/e37a796fc5d2866c2a6f95bcef98d02417ff4d17)

File: src/components/ToolMenu/Draw/DeleteAllButton/index.tsx (hunk 1 above)
Comment (faouziH21) [reply src/components/ToolMenu/Draw/DeleteAllButton/index.tsx]: done in:[36df83f](https://github.com/terrestris/shogun-gis-client/pull/1476/commits/36df83fa949184301bcbc5a0c8350cf936547cd6)

File: src/components/ToolMenu/Draw/index.tsx (hunk 3 above)
Comment (faouziH21) [reply src/components/ToolMenu/Draw/index.tsx]: done in: [36df83f](https://github.com/terrestris/shogun-gis-client/pull/1476/commits/36df83fa949184301bcbc5a0c8350cf936547cd6)

File: src/components/ToolMenu/Draw/DeleteAllButton/index.tsx (hunk 2 above)
Comment (faouziH21) [reply src/components/ToolMenu/Draw/DeleteAllButton/index.tsx]: done in: https://github.com/terrestris/shogun-gis-client/pull/1476/commits/28dbc8e5d678aa565d975ccdf1943ffa165c3b05

**Pull request merged**